privacyidea/lib/eventhandler/scripthandler.py

```diff
@@ -164,14 +186,23 @@ def do(self, action, options=None):
             proc_args.append("--logged_in_role")
             proc_args.append(logged_in_user.get("role", "none"))
 
+        rcode = 0
         try:
+            log.info("Starting script {script!r}.".format(script=script_name))
             p = subprocess.Popen(proc_args, cwd=self.script_directory)
-            log.info("Started script {script!r}:"
-                     " {process!r}".format(script=script_name, process=p))
+            if handler_options.get("background") == SCRIPT_WAIT:
+                rcode = p.wait()
+
         except Exception as e:
             log.warning("Failed to execute script {0!r}: {1!r}".format(
                 script_name, e))
             log.warning(traceback.format_exc())
 
+        if rcode:
+            log.warning("Script {script!r} failed to execute with error code {error!r}".format(script=script_name,
+                                                                                               error=rcode))
+            if handler_options.get("raise_error"):
```

Comment:
This is missing a ``is_true``, I think

Reply:
No. If raise error is set, then we will raise an error.
If it is not set, we will not raise an error - right? Or am I missing something?

Reply:
Thanks. Done in 83f6eaa